src/components/seo.js

```diff
@@ -25,32 +25,40 @@ function SEO({ description, lang, meta, keywords, title }) {
       meta={[
         {
           name: `description`,
-          content: metaDescription,
+          content: 'A place for Q&A and free mentorship',
+        },
+        {
+          property: `og:title`,
+          content: 'Mentor Night | Free Code Camp',
+        },
+        {
+          property: `og:description`,
+          content: 'A place to be coached or to coach for free along your developer journey',
+        },
+        {
+          property: `og:type`,
+          content: `website`,
+        },
+        {
+          name: `twitter:card`,
+          content: `summary`,
+        },
+        {
+          name: `twitter:creator`,
+          content: '@freeCodeCamp,'
+        },
+        {
+          name:'url',
+          content:"https://nashvillefcc.github.io/mentor-night-site/", //Added line 52 and it's key:value pair because the content key was present in other examples
```

Comment:
This comment is better put in the PR and left out of the code. It's useful to know the _why_ during the review but it just clutters up the actual code if left in there forever.

Also, make sure to keep with consistent spacing and usage of single or double quotes (until automated tooling enforces it for you 😃)

```suggestion
          name: 'url',
          content: 'https://nashvillefcc.github.io/mentor-night-site/',
```

Reply:
Agreed with itzsaga here. I'm also curious where you found the "url" tag from. I don't see it in the Twitter Card or OG specifications.